src/gl-matrix.js

```diff
@@ -26,12 +26,38 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE. */
 // END HEADER
 
-exports.glMatrix = require("./gl-matrix/common.js");
-exports.mat2 = require("./gl-matrix/mat2.js");
-exports.mat2d = require("./gl-matrix/mat2d.js");
-exports.mat3 = require("./gl-matrix/mat3.js");
-exports.mat4 = require("./gl-matrix/mat4.js");
-exports.quat = require("./gl-matrix/quat.js");
-exports.vec2 = require("./gl-matrix/vec2.js");
-exports.vec3 = require("./gl-matrix/vec3.js");
-exports.vec4 = require("./gl-matrix/vec4.js");
\ No newline at end of file
+import * as glMatrix from "./gl-matrix/common.js";
+import * as mat2 from "./gl-matrix/mat2.js";
+import * as mat2d from "./gl-matrix/mat2d.js";
+import * as mat3 from "./gl-matrix/mat3.js";
+import * as mat4 from "./gl-matrix/mat4.js";
+import * as mat4scalar from "./gl-matrix/mat4scalar.js";
+import * as mat4simd from "./gl-matrix/mat4simd.js";
+import * as quat from "./gl-matrix/quat.js";
+import * as vec2 from "./gl-matrix/vec2.js";
+import * as vec3 from "./gl-matrix/vec3.js";
+import * as vec4 from "./gl-matrix/vec4.js";
+
+mat4.scalar = {
```

Comment:
Interesting. This is one part of ES modules that I have not fully understood yet but your link will be helpful.